Configuration/webapp/app/routes/runs/details.tsx

```diff
@@ -21,15 +21,15 @@ import { RunLogs } from '~/components/runs/run-logs';
 import { Spinner } from '~/ui/spinner';
 
 const getRun = async (runNumber: number): Promise<Run> => {
-  const response = await fetch(`http://localhost:8080/api/runs/${runNumber}`);
+  const response = await fetch(`http://localhost:8080/control/api/runs/${runNumber}`);
```

Comment:
It is best to keep only the path and parameters to the request instead of the host and port full path. 
For example, this will work the same
```js
const response = await fetch(`control/api/runs/${runNumber}`);
```